src/mscorlib/corefx/System/Globalization/TextInfo.cs

```diff
@@ -38,38 +36,28 @@ public partial class TextInfo : ICloneable, IDeserializationCallback
         ////
 
         [OptionalField(VersionAdded = 2)]
-        private String m_listSeparator;
```

Comment:
> people already started to log issues against it

Isn't that a different serialization mechanism?

> I believe we'll run into scenarios with the cross-plat support

I've no doubt there are scenarios for it; I'm simply stating it's a huge amount of work with a big bug tail, and those scenarios aren't the prime ones we're currently aiming to enable (e.g. saving state from a game for a subsequent run).  My understanding is that Mono basically gave up on such cross-platform serialization as well, at least for higher-level types.